code/__HELPERS/tts.dm

```diff
@@ -0,0 +1,4 @@
+/proc/tts_alphanumeric_filter(text)
+	// Only allow alphanumeric characters and whitespace
+	var/static/regex/bad_chars_regex = regex("\[^a-zA-Z0-9 ,?.!'&-\\(\\)]", "g")
```

Comment:
yeah uh why the "g" it's not called repeatedly on the same text

Reply:
I want it to replace all bad matches, not just the first one

Reply:
ah I see, sure